website/docs/docs/cloud/migration.md

```diff
@@ -12,26 +12,38 @@ There is some preparation required to ensure a successful migration.
 3. **User invitations** &mdash; Any pending user invitations will be invalidated during the migration. You can resend the invitations once the migration is complete.
-4. **Git integrations** &mdash; Native integrations with [GitLab](/docs/cloud/git/connect-gitlab#for-the-dbt-cloud-enterprise-tier) and [Azure DevOps](/docs/cloud/git/connect-azure-devops) will need to be manually updated. dbt Labs will not be migrating any accounts using these integrations at this time. If you're using one of these integrations and your account is scheduled for migration, please contact support and we will delay your migration.
-5. **SSO integrations** &mdash; Integrations with SSO identity providers (IdPs) will need to be manually updated. dbt Labs will not be migrating any accounts using SSO at this time. If you're using one of these integrations and your account is scheduled for migration, please contact support and we will delay your migration.
 
-## Post-migration
+## Post-migration checklist
+
+Some integrations with third-party platforms can only be updated after migration is complete and your new access URL has been issued.
+
+:::info
+Accounts with integrations that depend on the new access URL are not currently targeted for migration.
 
-After migration, if you completed all the [Pre-migration checklist](#pre-migration-checklist) items, your dbt Cloud resources and jobs will continue to work as they did before. 
+If you're using an impacted integration and your account is scheduled for migration, please contact support and we will delay your migration.
+:::
+
+1. **Certain Git integrations** &mdash; Native integrations with [GitLab](/docs/cloud/git/connect-gitlab#for-the-dbt-cloud-enterprise-tier) and [Azure DevOps](/docs/cloud/git/connect-azure-devops) will need to be manually updated to reflect your new access URL. Accounts using GitHub or direct SSH connections do not require any updates.
+2. **SSO integrations** &mdash; Integrations with SSO identity providers (IdPs) will need to be manually updated to reflect your new access URL.
+
+## Accessing dbt Cloud after the migration
+
+After completing necessary pre- and post-migration checklist items, your dbt Cloud resources and jobs will continue to work as they did before.
 
 You have the option to log in to dbt Cloud at a different URL:
- * If you were previously logging in at `cloud.getdbt.com`, you should instead plan to login at `us1.dbt.com`. The original URL will still work, but you’ll have to click through to be redirected upon login.
- * You may also log in directly with your account’s unique [access URL](https://next.docs.getdbt.com/docs/cloud/about-cloud/access-regions-ip-addresses#accessing-your-account).
+
+* If you were previously logging in at `cloud.getdbt.com`, you should instead plan to login at `us1.dbt.com`. The original URL will still work, but you’ll have to click through to be redirected upon login.
```

Comment:
Will _all_ traffic drained from cloud.getdbt.com wind up on the `us1` cell? I imagine that's true in the short term, but surely at some point it will need to be spread across multiple cells. Should we be explaining how to find your true cell?